peak2bw.py

```diff
@@ -70,19 +70,18 @@ def main():
 
     # Read input files
     _logger.info('Reading input files')
-    peaks = pd.read_csv(args.input, sep='\t', header=None,
-                        usecols=[0, 1, 2], skiprows=args.skip)
-    sizes = pd.read_csv(args.sizes, sep='\t', header=None)
+    peaks = read_intervals(args.input, skip=args.skip)
+    sizes = read_sizes(args.sizes)
 
     # Subset to peaks in sizes file
     # TODO: Move this test into the df_to_bedGraph function
```

Comment:
is this TODO still relevant ? If so, can we make this change in this PR and remove the TODO ?

Reply:
Made the change.